Doc/reference/expressions.rst

```diff
@@ -523,7 +523,7 @@ is already executing raises a :exc:`ValueError` exception.
 .. method:: generator.__next__()
 
    Starts the execution of a generator function or resumes it at the last
-   executed yield expression.  When a generator function is resumed with a
```

Comment:
I think the original "a" is correct here, although I realize there's some wiggle room.